CHANGELOG.md

```diff
@@ -28,9 +38,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Development dependencies install via pip: `pip install "merlinwf[dev]"`.
 - `merlin status <yaml spec>` that returns queues, number of connected
-  workers and number of unused tasks in each of those queues.
-- `merlin example` cli command, which allows users to start running the
-  examples immedately (even after pip-installing).
```

Comment:
Don't think we want to delete these entries, looks like a merge mistake.

Reply:
OK, I'll look at his.

Reply:
Should be fixed now.